bitcoin/src/blockdata/transaction.rs

```diff
@@ -399,6 +402,20 @@ impl TxOut {
     }
 }
 
+#[cfg(feature = "arbitrary")]
+impl<'a> Arbitrary<'a> for TxOut {
+    fn arbitrary(u: &mut Unstructured<'a>) -> Result<Self> {
+        let value = Amount::arbitrary(u)?;
+        let script_pubkey = ScriptBuf::arbitrary(u).unwrap();
+        let tx_out = TxOut {
+            value,
+            script_pubkey
+        };
+
+        Ok(tx_out)
```

Comment:
Any reason this is not written like this (and returning the error instead of using `unwrap`)

```rust
    fn arbitrary(u: &mut Unstructured<'a>) -> Result<Self> {
        Ok(TxOut {
            value: Amount::arbitrary(u)?,
            script_pubkey: ScriptBuf::arbitrary(u)?,
        })
```

Reply:
I sometimes find nesting makes things hard to read, even though it's more compact.  This is fine though.

Reply:
That reminds me of an easter egg in the python repl

```
>>> import this
The Zen of Python, by Tim Peters

Beautiful is better than ugly.
Explicit is better than implicit.
Simple is better than complex.
Complex is better than complicated.
Flat is better than nested.
Sparse is better than dense.
Readability counts.
Special cases aren't special enough to break the rules.
Although practicality beats purity.
Errors should never pass silently.
Unless explicitly silenced.
In the face of ambiguity, refuse the temptation to guess.
There should be one-- and preferably only one --obvious way to do it.
Although that way may not be obvious at first unless you're Dutch.
Now is better than never.
Although never is often better than *right* now.
If the implementation is hard to explain, it's a bad idea.
If the implementation is easy to explain, it may be a good idea.
Namespaces are one honking great idea -- let's do more of those!
```